src/Spryker/Zed/AppPayment/Business/AppPaymentFacadeInterface.php

```diff
@@ -78,13 +80,36 @@ public function getPaymentPage(PaymentPageRequestTransfer $paymentPageRequestTra
      */
     public function handleWebhook(WebhookRequestTransfer $webhookRequestTransfer, WebhookResponseTransfer $webhookResponseTransfer): WebhookResponseTransfer;
 
+    /**
+     * Specification:
+     * - Requests the `AppPaymentPlatformPaymentMethodsPluginInterface::configurePaymentMethods()` method to return a list PaymentMethods to be added.
+     * - When the passed `AppPaymentPlatformPluginInterface` is not an instance of `AppPaymentPlatformPaymentMethodsPluginInterface` it will return early.
+     * - PaymentMethods that were already added will not be added again.
+     * - PaymentMethods that were already persisted and are no longer returned from `AppPaymentPlatformPaymentMethodsPluginInterface::configurePaymentMethods()` method will be deleted and trigger a `DeletePaymentMethod` message.
+     * - PaymentMethods that were already persisted and require an update (payment method data has changed) will trigger a `UpdatePaymentMethod` message.
+     *
+     * @api
+     */
+    public function configurePaymentMethods(AppConfigTransfer $appConfigTransfer): AppConfigTransfer;
+
+    /**
+     * Specification:
+     * - Requests the `AppPaymentPlatformPaymentMethodsPluginInterface::configurePaymentMethods()` method to return a list PaymentMethods to add and to delete.
```

Comment:
```suggestion
     * - Requests the `AppPaymentPlatformPaymentMethodsPluginInterface::configurePaymentMethods()` method to return a list PaymentMethods to delete.
```
what if some of the payment methods were not selected by a user from the whole list of `configurePaymentMethods()`, will it "delete" all methods anyway?

Reply:
check it again, it's spec for `deletePaymentMethods()`